apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreWallet/RestoreWalletScreen.tsx

```diff
@@ -57,21 +128,53 @@ export const RestoreWalletScreen = () => {
             title={strings.next}
             style={styles.button}
             disabled={isEmptyString(mnemonic)}
-            onPress={async () => {
-              const {accountPubKeyHex} = await makeKeys({mnemonic})
-              mnemonicChanged(mnemonic)
-              publicKeyHexChanged(accountPubKeyHex)
-              navigation.navigate('setup-wallet-restore-details')
-            }}
+            onPress={saveMnemonic}
           />
-
-          <Space height="s" />
         </View>
       </KeyboardAvoidingView>
     </SafeAreaView>
   )
 }
 
+const Modal = ({
+  onPress,
+  accountPlate,
+  walletName,
+}: {
+  onPress: () => void
+  accountPlate: WalletChecksum
+  walletName: string
+}) => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+
+  return (
+    <View style={{flex: 1}}>
```

Comment:
isn't the safeareaview needed?

Reply:
my bet is the last spacer is doing the job of safearewview without the navbarbottom

Reply:
there is ticket for that

Reply:
that view is for the Modal